Misc/python-config.sh.in

```diff
@@ -4,7 +4,9 @@
 
 exit_with_usage ()
 {
-    echo "Usage: $0 --prefix|--exec-prefix|--includes|--libs|--cflags|--ldflags|--extension-suffix|--help|--abiflags|--configdir|--embed"
+    echo "Usage: $0 --prefix|--exec-prefix|--includes|--libs|--cflags|\
+        --ldflags|--extension-suffix|--help|--abiflags|--configdir|--embed|\
+        --version|--machdep|--multiarch"
```

Comment:
Can we please create a separated PR for this change which doesn't seem to be directly related?

Reply:
As explained in my first comment, this is used to get the configuration variables needed to import the sysconfigdata module, see `_get_python_config()` in Lib/sysconfig.py in the PR changes.